workflows/process/prepare_results/seqkit_tool_agreements.nf

```diff
@@ -3,7 +3,8 @@ process seqkit_tool_agreements {
 /*         errorStrategy 'ignore' */
         label 'seqkit'
     input:
-        tuple val(name), path(toolagreement_per_contig), path(fasta)
+        tuple val(name), path(toolagreement_per_contig)
+        tuple val(name), path(fasta)
```

Comment:
how does this work.? what happens when multiple samples are used? nextflow is not joining them by correct name?